### PR TITLE
[feat ] First version of graphql client working

### PR DESCRIPTION
**Pull request by tshuck**
@sisk @DLavin23 

First version of react working with graphql.  You can click on songs and they get created!!!!!

Decided to use a fairly [slim implementation](https://www.npmjs.com/package/graphql.js) instead of diving right in to apollo.

### CODE REVIEW

File: src/Room/index.tsx [hunk 1]
```diff
@@ -41,26 +41,28 @@ class Room extends React.Component<Props, {}> {
       songs,
       results,
     } = this.props
-
-    const player = songs.length > 0 ? <ReactPlayer url={songs[songs.length - 1].url} playing={true} /> : ''
+    console.log(songs)
```
Comment (DLavin23) [inline src/Room/index.tsx]: do we still need this `console.log` statement?

File: src/Room/index.tsx [hunk 2]
```diff
@@ -41,26 +41,28 @@ class Room extends React.Component<Props, {}> {
...
       return (
         <li key={result.id}>
           <div onClick={onClick}>
             {result.title} - {result.description}
+            <img src={result.image} />
           </div>
         </li>
       )
     })
+    const floop = () => this.props.createSong({ youtubeId: 'KFTz3a-kSj4' }, '', '')
```
Comment (DLavin23) [inline src/Room/index.tsx]: is `floop` short for something?

File: src/models/song/sagas.ts [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
 
-  const roomId = '66cc51aa-83e0-4880-a84e-b69d4a5507ed'
+  // const roomId = '66cc51aa-83e0-4880-a84e-b69d4a5507ed'
   const api = getSingleton()
-  const data = withRoom(roomId, songSerializer(action.song))
-  yield apply(api, api.rooms.id(roomId).song.create, [{data}])
+
+  yield apply(api, api.songs.create, [action.song.youtubeId])
+  // const data = withRoom(roomId, songSerializer(action.song))
+  // yield apply(api, api.rooms.id(roomId).song.create, [{data}])
```
Comment (DLavin23) [inline src/models/song/sagas.ts]: do we still need these comments?

**Review by DLavin23 — APPROVED** [review]
couple of small comments, otherwise LGTM

**Comment by tshuck** [comment]
@DLavin23 `floop` stands for "**F**rick, I'd **l**ove it if I c**o**uld get this graph ql thing to w**o**rk, **p**lease!"  PR contains a bunch of nonsense as you rightly pointed out; I'll clean this up a bit to match the recent work over in the api.

**Comment by tshuck** [comment]
@DLavin23 

Added a big ol' commit that cleans some stuff up + gets some of the CRUD actions working with graphql.  Next up is to re-integrate action cable, add the room queue.

musicbox

**Pull request merged**